libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4261,7 +4290,11 @@ MAV_RESULT GCS_MAVLINK::handle_command_long_packet(const mavlink_command_long_t
     case MAV_CMD_FIXED_MAG_CAL_YAW:
         result = handle_fixed_mag_cal_yaw(packet);
         break;
-        
+
+    case MAV_CMD_CONTROL_HIGH_LATENCY:
+        result = handle_command_control_high_latency(packet);
```

Comment:
Pretty sure `SET_MESSAGE_INTERVAL` makes this message redundant.

Reply:
``SET_MESSAGE_INTERVAL`` will indeed have the same effect. https://mavlink.io/en/services/high_latency.html appears to suggest that ``MAV_CMD_CONTROL_HIGH_LATENCY`` is a specific command for stopping and starting the HIGH_LATENCY2 messages.

That's how QGC has implemented it anyway - it deliberately sends a MAV_CMD_CONTROL_HIGH_LATENCY and waits for the COMMAND_ACK before dropping into "high latency" mode.

Reply:
Fixed to use ``SET_MESSAGE_INTERVAL``